<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>764</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>776</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton#resize_button {
	border-style: solid;
	border-width: 1px;
	border-color: rgb(85, 170, 255);
	border-radius: 8px;

}

QPushButton#resize_button::hover {
	border-color: rgb(255, 170, 255);
	background-color: rgb(218, 218, 218);
}

QPushButton {
	border-style: solid;
	border-width: 1px;
	border-color: rgb(85, 170, 255);
	border-radius: 5px;

}

QPushButton::hover {
	border-color: rgb(255, 170, 255);
	background-color: rgb(218, 218, 218);
}

QPushButton::pressed {
	border-color: rgb(255, 170, 255);
	background-color: rgb(220, 220, 220);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="PictureArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color : rgb(70, 71, 71); color : rgb(252, 255, 255); font: 57 25pt &quot;Consolas&quot;;}

</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Please select a picture first&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PictureHistogram">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {

background-color : rgb(70, 71, 71); 
color : rgb(252, 255, 255); 
font: 57 15pt &quot;Consolas&quot;;
}

</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Please select a picture first&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>511</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 245, 245);

</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Basic">
     <attribute name="title">
      <string>基本功能</string>
     </attribute>
     <widget class="QPushButton" name="resize_button">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>3</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>图像压缩</string>
      </property>
      <property name="iconSize">
       <size>
        <width>15</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="sobel_picture">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string> 图像锐化</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="title">
       <string>亮度调整</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>20</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="title">
       <string>对比度调整</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QSlider" name="ContrastSlider">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>20</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-50</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="blur_picture">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>103</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string> 图像模糊</string>
      </property>
     </widget>
     <widget class="QPushButton" name="floodfill">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>125</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>漫水填充</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rotate_picture">
      <property name="geometry">
       <rect>
        <x>358</x>
        <y>80</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>图像旋转</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gray_picture">
      <property name="geometry">
       <rect>
        <x>358</x>
        <y>103</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>转为灰度图</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cut_pic">
      <property name="geometry">
       <rect>
        <x>358</x>
        <y>125</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>图像裁剪</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="pic_size">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Apple LiGothic</family>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Apple LiGothic'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QSlider" name="resize_coefficient">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>30</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;倍率&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox_2">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>45</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">min-width: 65;
min-height: 20;</string>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>72</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>7</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 7pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;当前大小&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="contrast_button">
      <property name="geometry">
       <rect>
        <x>352</x>
        <y>45</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">min-width: 65;
min-height: 20;</string>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>164</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="RGB_R">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RGB_G">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RGB_B">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>105</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>图层通道</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Advanced">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>高级功能</string>
     </attribute>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">min-width: 70;
min-height: 20;
font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QSlider" name="MixSlider">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>84</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>50</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>混合率(1/2)</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>163</width>
        <height>40</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;图片混合&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pic_mix">
         <property name="styleSheet">
          <string notr="true">min-height: 20px;
font: 8pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>选择图片2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>160</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="styleSheet">
          <string notr="true">min-width: 65;
min-height: 25;
font: 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>生成鱼眼图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">min-width: 65;
min-height: 25;
font: 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>人脸检测</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="thr_binary">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>38</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>二值化阈值</string>
      </property>
     </widget>
     <widget class="QPushButton" name="thr_inv_bin">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>68</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>反二值化阈值</string>
      </property>
     </widget>
     <widget class="QPushButton" name="thr_trunc">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>38</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>截断阈值</string>
      </property>
     </widget>
     <widget class="QPushButton" name="thr_to_zero">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>38</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>阈值化为0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="thr_to_zero_inv">
      <property name="geometry">
       <rect>
        <x>363</x>
        <y>68</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>反阈值化为0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cam_camshift">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>物体追踪</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cam_optiflow">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>人脸检测</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cam_facedetect">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>光流算法</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>160</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>图片阈值化</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="ThresholdSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>328</x>
        <y>98</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>摄像头功能</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="History">
     <attribute name="title">
      <string>修改历史</string>
     </attribute>
     <widget class="QTextEdit" name="history">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>121</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>160</y>
      <width>811</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
